### PR TITLE
Add videos

### PR DESCRIPTION
**Pull request by AtulPant2704**
- Add videos and categories to backend
- Created Context for videos and fetched using network call
Preview Link:- https://deploy-preview-3--checkmate-stream.netlify.app/explore

### CODE REVIEW

**Review by Kevin-Solomon — APPROVED** [review]
Nice Work.
Clean Modular Code.

File: src/pages/VideoListing/Explore.jsx [hunk 1]
```diff
@@ -15,21 +18,9 @@ const Explore = () => {
...
-                        <VideoCard />
-                        <VideoCard />
-                        <VideoCard />
-                        <VideoCard />
-                        <VideoCard />
-                        <VideoCard />
-                        <VideoCard />
-                        <VideoCard />
+                        {videos.map(video => (
+                            <VideoCard key={video._id} {...video} />
```
Comment (Ayushi20-19) [inline src/pages/VideoListing/Explore.jsx:22]: like this approach for sending data using spread

**Review by Ayushi20-19 — APPROVED** [review]
-Like the file extraction for services, and API calls
-Clean code.

**Review by ChhikaraBRUH — APPROVED** [review]
Nice Work Atul. Well written code.

**Pull request merged**